deploy/jbossas/kit/jboss-wf8/domain/configuration/modeshape/sample-ha-cache-config.xml

```diff
@@ -1,76 +1,68 @@
-                    shared="false"
-                    fetchPersistentState="true"
-                    purgeOnStartup="true"
-                    location="${jboss.server.data.dir}/modeshape/store/sample-ha">
-            </singleFile>
-        </persistence>
-    </namedCache>
-
-    <namedCache name="sample-binary-fs">
-        <clustering mode="replication">
-            <sync />
-        </clustering>
-        <locking lockAcquisitionTimeout="60000"/>
-        <transaction
-                transactionManagerLookupClass="org.infinispan.transaction.lookup.GenericTransactionManagerLookup"
-                transactionMode="TRANSACTIONAL"
-                lockingMode="PESSIMISTIC"/>
-        <persistence
-                passivation="false">
-            <singleFile
-                    shared="false"
-                    preload="false"
-                    fetchPersistentState="false"
-                    purgeOnStartup="false"
-                    location="${jboss.server.data.dir}/modeshape/binary-store/sample-ha-binary-fs">
-            </singleFile>
-        </persistence>
-    </namedCache>
-
-    <namedCache name="sample-binary-fs-meta">
-        <clustering mode="replication">
-            <sync />
-        </clustering>
-        <locking lockAcquisitionTimeout="60000"/>
-
-        <transaction
-                transactionManagerLookupClass="org.infinispan.transaction.lookup.GenericTransactionManagerLookup"
-                transactionMode="TRANSACTIONAL"
-                lockingMode="PESSIMISTIC"/>
-        <persistence
-                passivation="false">
-            <singleFile
-                    shared="false"
-                    preload="false"
-                    fetchPersistentState="false"
-                    purgeOnStartup="false"
-                    location="${jboss.server.data.dir}/modeshape/binary-store/sample-ha-binary-fs-meta">
-            </singleFile>
-        </persistence>
-    </namedCache>
-</infinispan>
+            xsi:schemaLocation="urn:infinispan:config:7.0 http://www.infinispan.org/schemas/infinispan-config-7.0.xsd"
+            xmlns="urn:infinispan:config:7.0">
+    <jgroups>
+        <stack-file name="tcp" path="${jboss.domain.config.dir}/modeshape/jgroups-config.xml"/>
+    </jgroups>
+    <cache-container default-cache="sample" statistics="false">
+        <jmx duplicate-domains="true"/>
+        <transport
+                cluster="modeshape-cluster"
+                stack="tcp"/>
+        <!-- 
+            Repo cache
+        -->
+        <replicated-cache name="sample" mode="SYNC">
+            <locking
+                    isolation="READ_COMMITTED"
+                    acquire-timeout="1000"/>
+            <transaction
+                    mode="NON_XA"
+                    locking="PESSIMISTIC"/>
+            <persistence>
+                <file-store
+                        shared="false"
+                        preload="false"
+                        purge="true"
```

Comment:
yes, if neither `preload` nor `purge` are set to `true` ISPN issues an ugly warning in replicated mode.